services-core/src/driver/stablex_driver.rs

```diff
@@ -35,7 +35,7 @@ pub struct StableXDriverImpl {
     price_finder: Arc<dyn PriceFinding + Send + Sync>,
     orderbook_reader: Arc<dyn StableXOrderBookReading>,
     solution_submitter: Arc<dyn StableXSolutionSubmitting + Send + Sync>,
-    economic_viability: Arc<dyn EconomicViabilityComputing + Send + Sync>,
+    economic_viability: Arc<dyn EconomicViabilityComputing>,
```

Comment:
Noob question: What is the difference now without `Send` and `Sync` and why did we need them before, but not now?

Reply:
Good question. On the trait `EconomicViabilityComputing` we already had the requirement `Send + Sync` (which means that it is safe to use from multiple threads). So even before this PR, specifying this requirement here again wasn't necessary. I noticed that while making this change so included it.